README.md

```diff
@@ -42,6 +42,8 @@ $ docker-compose run cassandra-website
 
 :warning: *Tip:* In order to prevent root-owned modified files in your repository, the container user, `build`, is set up with a default UID=1000:GID=1000, which is usually the first user configured on a linux machine. If your local user is different you should set up the container user with your local host user's UID:GID, replace the above with:
 
+:warning: *Tip:* Building of cassandra-website may fail on not having `CLOUDSDK_PYTHON` variable set at your machine. Set that environment variable for example like `export CLOUDSDK_PYTHON=/usr/bin/python2`.
```

Comment:
Building cassandra-website may fail if the `CLOUDSDK_PYTHON` environment variable is not set on your machine. For example, set the environment variable to `export CLOUDSDK_PYTHON=/usr/bin/python2`.